sprint-guide/exercises/swot-analysis.md

```diff
@@ -44,7 +44,7 @@ Weaknesses are things that we consider the company or product to be lacking in o
 - What is something that performs badly?
 - What do the target audience complain about or criticise?
 
-For example: "Long waiting times", "lack of persoanlity", "Users complain about lost items" are all weaknesses.
+For example: "Long waiting times", "lack of personality", "Users complain about lost items" are all weaknesses.
```

Comment:
I am not sure I understand the capitalization rules used for the examples – they appear inconsistent to me. Shouldn’t it be “Lack of personality” instead?

Reply:
Yeah, standardizing the capitalization by capitalizing that L would be an improvement. Thanks.

Reply:
Great!

The examples in other sections follow a similar pattern. I capitalized them. In addition, the lead-in sentence for each question list felt repetitive and “these” was used twice referring to different subjects.

Let me know, if a change does not align and I'll drop it.

Reply:
This looks good, thanks for your contribution!